### PR TITLE
Added host networking as a service option

### PR DESCRIPTION
**Pull request by loganintech**

### CODE REVIEW

File: charts/factorio/values.yaml [hunk 1]
```diff
@@ -47,22 +47,28 @@ nodeSelector: {}
...
+  type: NodePort
+  port: 31497
+  ## set the port to a value you like in the range of 30000-32767. Leave it blank for a random port
+  # nodePort: ""
+
+  ## Host networking setup
+  ## This is the most limiting setup, using the networking of the k8s host.
+  ## Whatever port the server uses (34197) is what will be used
+  ## If you want to have multiple servers running on the same host, use NodePorts instead
+  #hostNetwork: true
```
Comment (tobimichael96) [inline charts/factorio/values.yaml:66]: What happens if you enable hostNetwork and have the NodePort in place as well? Does that work out?

Comment (loganintech) [reply charts/factorio/values.yaml:66]: It'll use host networking and your NodePort settings will be ignored.

Comment (tobimichael96) [reply charts/factorio/values.yaml:66]: Cool, thanks! :)

**Comment by loganintech** [comment]
@tobimichael96 Gave you credit for the release https://github.com/loganintech/helm-charts/releases/tag/factorio-1.2.7

You can use 1.2.7 now in your helm charts.

**Pull request merged**